ui/main/src/app/modules/externaldevicesconfiguration/editModal/signal-mappings-modal.component.html

```diff
@@ -0,0 +1,81 @@
+      <of-spinner *ngIf="isLoading()" loadingText="shared.loadingInProgress"></of-spinner> 
+
+      <div *ngIf="!isLoading()">
+        <form [formGroup]="signalMappingsForm" class="opfab-form-lighter">
+          <div class="container">
+            <div class="col">
+
+              <div *ngIf="row" class="opfab-input opfab-edit-row">
+                <label for="opfab-signalMappingId" translate>externalDevicesConfiguration.signalMapping.id</label>
+                <input type="text" disabled value="{{row.id}}"/>
+              </div>
+              <div *ngIf="!row"  class="opfab-input opfab-edit-row">
+                <label for="opfab-deviceId" translate>externalDevicesConfiguration.signalMappings.id</label>
+                <input formControlName="id" name="signalMappingId" id="opfab-signalMappingId" type="text"/>
+              </div>
+              <div *ngIf="id.invalid" class="row alert alert-danger">
+                <div *ngIf="id.errors.required" translate>admin.input.id.required</div>
+                <div *ngIf="id.errors.uniqueDeviceIdViolation" translate>admin.input.id.alreadyExisting</div>
+              </div>
+              <div class="opfab-input opfab-edit-row">
+                <label for="opfab-signal-mapping-alarm" translate>shared.severity.alarm</label>
+                <input formControlName="alarm" name="alarm" id="opfab-signal-mapping-alarm" type="number"/>
+              </div>
+              <div *ngIf="alarm.invalid" class="row alert alert-danger">
+                <div *ngIf="alarm.errors.required" translate>externalDevicesConfiguration.input.signalMapping.required</div>
+              </div>
+              <div class="opfab-input opfab-edit-row">
+                <label for="opfab-signal-mapping-action" translate>shared.severity.action</label>
+                <input formControlName="action" name="alarm" id="opfab-signal-mapping-action" type="number"/>
+              </div>
+              <div *ngIf="action.invalid" class="row alert alert-danger">
+                <div *ngIf="action.errors.required" translate>externalDevicesConfiguration.input.signalMapping.required</div>
+              </div>
+              <div class="opfab-input opfab-edit-row">
+                <label for="opfab-signal-mapping-information" translate>shared.severity.information</label>
+                <input formControlName="information" name="information" id="opfab-signal-mapping-information" type="number"/>
+              </div>
+              <div *ngIf="information.invalid" class="row alert alert-danger">
+                <div *ngIf="information.errors.required" translate>externalDevicesConfiguration.input.signalMapping.required</div>
+              </div>
+              <div class="opfab-input opfab-edit-row">
+                <label for="opfab-signal-mapping-compliant" translate>shared.severity.compliant</label>
+                <input formControlName="compliant" name="compliant" id="opfab-signal-mapping-compliant" type="number"/>
```

Comment:
Switch information and compliant in the edit modal to be consistent with the display